src/Components/Authentication/__tests__/Mobile/LoginForm.test.tsx

```diff
@@ -52,7 +52,7 @@ describe("MobileLoginForm", () => {
     })
   })
 
-  it("renders password error", () => {
+  xit("renders password error", () => {
```

Comment:
The two following tests seem pretty critical, why `xit`?

Reply:
I'd love to figure out a way to get this working but I don't think these Formik actions are as accessible as they once were. 

I get the following error from these two specs which try to call `setStatus` directly

```
TypeError: Cannot read property 'setStatus' of null
```

Reply:
Maybe instead of 
```
const formik: any = wrapper.find("Formik").instance()
```
we could dive directly into the props like
```
const formik: any = wrapper.props().formik
```
(or some similar thing?)

Reply:
Just re-enabled these specs with simulated form entry. Seems verbose but better than skipping for sure.